krod/core/llm_manager.py

```
@@ -182,42 +182,98 @@ async def _call_openai_chat(
     async def generate_text(
         self,
         prompt: str,
-        model: str = "gpt-3.5-turbo",
+        model: str = "gpt-3.5-turbo" or "gpt-4o",
```

Ruff (0.11.9)

Use `"gpt-3.5-turbo"` instead of `"gpt-3.5-turbo" or ...`

Replace with `"gpt-3.5-turbo"`

(SIM222)